STEER/STEER/AliLHCDipValT.h

```diff
@@ -117,9 +117,9 @@ Char_t* AliLHCDipValT<Element>::TimeAsString(double t, Bool_t utc)
   static char buff[22];
   time_t tt = (time_t) t;
   struct tm *time = utc ? gmtime(&tt) : localtime(&tt);
-  snprintf(buff,21,"%02d:%02d:%02d %02d/%02d/%04d",time->tm_hour,time->tm_min,time->tm_sec,
-	  time->tm_mday,time->tm_mon+1,time->tm_year+1900);
-  return (char*)buff;
+  if (snprintf(buff,21,"%02d:%02d:%02d %02d/%02d/%04d",time->tm_hour,time->tm_min,time->tm_sec,
+	  time->tm_mday,time->tm_mon+1,time->tm_year+1900) ) return (char*)buff;
```

Comment:
Hi @zwound40 , under which circumstances this snprinf can return 0? As far as I see it can never happen

Reply:
Hi @shahor02 , sorry this was a failure on my side. sprintf will return a negative number in case of failure, not 0. I committed a fix.

Reply:
@zwound40 OK, thanks, this makes more sense. But did you encounter a situation when the snprintf has failed?

Reply:
@shahor02 : No, the only reason for this fix it that gcc (tested with gcc 8.0.2) gives a format-truncation warning, if the return value is not used. Since the code is by default compiled with -Werror this results in the compilation failing.  One could maybe even do something like:
int dummy = sprintf(...);
return return (char*)buff;
with a corresponding comment in the code that this is simply a hack to make gcc happy.

Reply:
OK, thanks.